### PR TITLE
Feat: text utilities

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Added headings and texts

Files to check:
[Text utilities](https://github.com/Muzammil-khan710/Component-Library/pull/9/files?file-filters%5B%5D=.css&file-filters%5B%5D=.html&show-viewed-files=true)

### CODE REVIEW

File: Component-Library-dev/docs/components/text-utilities/text-util.css [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    font-size: 1.5rem;
+}
+
+.heading-six {
+    font-size: 1.2rem;
+}
+
+.small-text {
+    font-size:0.9rem
+}
```
Comment (Shubhamkr2610) [inline Component-Library-dev/docs/components/text-utilities/text-util.css]: suggestion: you can give values in pixel for less than 1 rem

**Review by Shubhamkr2610 — COMMENTED** [review]
everything looks good to me . your code looks clean and readable . Easy to understand 
you can merge it

File: Component-Library-dev/docs/components/cards/cards.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
+.card-img {
+    height: 200px;
+    width: 100%;
+    border-radius: 5px 5px 0 0 ;
+}
+
+.card {
+    border: 1px solid black;
+    /* max-width: 600px; */
```
Comment (Shubhamkr2610) [inline Component-Library-dev/docs/components/cards/cards.css]: remove unused code

**Review by Shubhamkr2610 — APPROVED** [review]

**Review by iRohitGaur — APPROVED** [review]
Suggestion: Make smaller PRs if possible.

Nice work! Good to merge 👍🏻

**Pull request merged**